battleship.py

```diff
@@ -62,7 +62,13 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
+    matrix=[]
+    for i in range(rows):         
+        a =[]

```

Comment:
Maintain a proper naming conventions. Try to keep variables in such a way that they are readable.

Reply:
changed as per request